src/platform/libretro/libretro.c

```diff
@@ -96,6 +96,13 @@ static void _reloadSettings(void) {
 			mCoreConfigSetDefaultValue(&core->config, "idleOptimization", "detect");
 		}
 	}
+	
+	var.key = "mgba_frameskip";
+	var.value = 0;
+	if (environCallback(RETRO_ENVIRONMENT_GET_VARIABLE, &var) && var.value) {
+		opts.frameskip = strtol(var.value, NULL, 10);
+
```

Comment:
Was this extra newline in my original code?

Reply:
I think so:
https://github.com/mgba-emu/mgba/blob/master/src/platform/libretro/libretro.c#L95

Reply:
Oops